packages/currency/src/index.js

```diff
@@ -127,6 +127,7 @@ export default class Currency {
 /**
  * Returns currency data by country/region. Contains code, symbol, position, thousands separator, decimal separator, and precision.
  *
+ * Dev Note: When adding new currencies below, the exchange rate array should also be updated in WooCommerce Admin's `business-details.js`.
```

Comment:
Not opposed to the way you've done it here as that may be better, but wanted to get your opinion on adding the exchange rates directly to the items below for easy access:

```
		CA: {
			code: 'CAD',
			symbol: '$',
			symbolPosition: 'left',
			thousandSeparator: ',',
			decimalSeparator: '.',
			precision: 2,
            usdExchangeRate: 1.32
		},
```

This would be easier to maintain and may cut down some of the logic here.  Or are there some benefits to separating this out?

Reply:
My main reasoning was mentioned in the PR description above (which might be better as a comment):

> I didn't end up storing the exchange rates with the rest of the currency info. That info is in an exportable public currency package, and I worry about the rates getting used the wrong way if we expose them there. They are meant to be rough and not useful in most situations, so I opted to just include them in the conversion function, inside the dashboard.

If you're not that worried about it and think documentation is enough (or naming it something like `roughExchangeRate`) I'm happy to include it in the currency package though.

Reply:
Sorry, I must have skipped right over this above.

That makes a lot of sense to me.  Let's keep it as is.